sos/plugins/openstack_cinder.py

```diff
@@ -37,14 +37,22 @@ def setup(self):
             "systemctl status openstack-cinder-api.service"
         )
 
+        ps_status = self.get_command_output("ps -ef")
+        in_ps = False
+        if ps_status['status'] == 0:
+            for line in ps_status['output'].splitlines():
+                if "cinder_wsgi" in line:
+                    in_ps = True
+                    break
+
         container_status = self.get_command_output("docker ps")
         in_container = False
         if container_status['status'] == 0:
             for line in container_status['output'].splitlines():
                 if line.endswith("cinder_api"):
                     in_container = True
 
```

Comment:
Patch sounds OK, maybe trivial enhancement outside of its scope would be calling "break" here (like on line 46)?

Reply:
yes makes sense. I submitted a change with the additional "break"